github/ipld.yml

```
@@ -3226,108 +3146,41 @@ teams:
       maintainer:
         - rvagg
         - SgtPooki
-      member:
-        - hacdias
-        - lidel
-        - olizilla
-        - whizzzkid
-    privacy: closed
-  Java Team:
-    description: Java Java Java Java
-    members:
-      maintainer:
-        - jbenet
-      member:
-        - ianopolous
     privacy: closed
   JavaScript Team:
     description: The most awesome team
     members:
       maintainer:
-        - daviddias
-        - dignifiedquire
         - rvagg
         - vmx
       member:
         - achingbrain
-        - AdamStone
         - alanshaw
-        - Gozala
-        - haadcode
-        - hacdias
-        - hugomrdias
-        - ipldbot
-        - kumavis
-        - litzenberger
-        - magik6k
-        - mcollina
-        - momack2
-        - nicola
         - olizilla
-        - pkafei
-        - richardschneider
-        - tchardin
-        - vasco-santos
-        - victorb
-        - wanderer
-    privacy: closed
-  Python Team:
-    description: Snaaaaakes 🐍
-    members:
-      maintainer:
-        - daviddias
-      member:
-        - dhruvbaldawa
     privacy: closed
   research:
     members:
       maintainer:
-        - dignifiedquire
-        - jbenet
-        - miyazono
-        - nicola
-        - Stebalien
         - vmx
-      member:
-        - davidad
     privacy: closed
   reviewers:
     members:
       maintainer:
         - rvagg
-        - warpfork
       member:
         - BigLep
-        - RangerMauve
         - willscott
     privacy: closed
   Rust Team:
```

```suggestion
  Rust Maintainers
```

To be consistent and make clear there isn't a "Rust Team"?